cpp/src/DataStorm/Contract.ice

```diff
@@ -4,244 +4,284 @@
 #pragma once
 
 #include <Ice/Identity.ice>
+#include <Ice/BuiltinSequences.ice>
 #include <DataStorm/Sample.ice>
 
 [["cpp:include:deque"]]
 
 module DataStormContract
 {
+    enum ClearHistoryPolicy
+    {
+        OnAdd,
+        OnRemove,
+        OnAll,
+        OnAllExceptPartialUpdate,
+        Never
+    };
+
+    /// A dictionary of <long, long>
```

Comment:
That's not a good doc-comment.

Reply:
Removed, I would add more comments in a follow-up PR.